packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -159,9 +161,9 @@ export function WithdrawalConfirmationDialog(props: UseDialogProps) {
   const [checkbox2Checked, setCheckbox2Checked] = useState(false)
 
   const bothCheckboxesChecked = checkbox1Checked && checkbox2Checked
+  const confirmationDays = Math.floor(l1.network.blockTime * l2.network.confirmPeriodBlocks / SECONDS_IN_DAY)
 
-  const { isMainnet } = isNetwork(l1.network)
-  const confirmationPeriod = isMainnet ? '~8 days' : '~1 day'
+  const confirmationPeriod = `~${confirmationDays} day${confirmationDays > 1 ? 's' : ''}`
```

Comment:
Let's use `Math.round` and actually do something like:

```
if >= 2 days
  show number of days
else
  show number of hours
```

So we'll have `~8 days` for mainnets, and `~4 hours` and `~27 hours` for testnets.

Reply:
@spsjvc done